### PR TITLE
Adding initial api server sample using Zalando connexion framework

### PR DESCRIPTION
**Pull request by scoulomb**
- Reproducing issue with invalid JSON and content-type header when using Connexion 
- Demo sample

### CODE REVIEW

File: .flask_version/run.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/api/v1/test/entry", methods=["POST"])
+def test_entry():
+    content = request.get_json()
+    # to be fully equivalent we should manage mandatory params etc ... like this
```
Comment (Jiehong) [inline .flask_version/run.py:17]: doesn't make sense without context Remove this comment

File: .flask_version/run.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/api/v1/test/entry", methods=["POST"])
+def test_entry():
```
Comment (Jiehong) [inline .flask_version/run.py:15]: What about passing `body` instead?

File: documentation/openapi/openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  title: Minimal project with connexion
+  description: Sample project
+  version: 1.0.0
+servers:
+  - url: /api/v1/test
+paths:
+  /entry:
+    post:
+      operationId: sample_code.endpoints.endpoint.post
+      description: Declare a load balancer SI entry
```
Comment (Jiehong) [inline documentation/openapi/openapi.yaml:12]: remove, and simplify a bit more

File: .flask_version/run.py (hunk 1 above)
Comment (scoulomb) [reply .flask_version/run.py:17]: removed

File: .flask_version/run.py (hunk 2 above)
Comment (scoulomb) [reply .flask_version/run.py:15]: Seems not possible: https://stackoverflow.com/questions/10434599/get-the-data-received-in-a-flask-request

File: documentation/openapi/openapi.yaml (hunk 3 above)
Comment (scoulomb) [reply documentation/openapi/openapi.yaml:12]: Done

**Pull request merged**